lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -363,7 +367,7 @@ def _probabilities_to_percentiles(
             air_temperature at the required percentiles.
 
         """
-        threshold_coord = find_coordinate(forecast_probabilities, "threshold")
+        threshold_coord = forecast_probabilities.coord("threshold")
```

Comment:
I've kept the changes as they are, although yes, this needs to be revisited when the support for the probability to percentile conversion is implemented.